lib/jekyll-node-module/filters.rb

```diff
@@ -10,7 +12,7 @@ def node_module_url(input)
           site.static_files << file
         end
 
-        file.url
+        relative_url(file.url)
```

Comment:
> The more consistent approach may be what I recommended earlier, by chaining filters in your template.
True, but then you have to know to do this, and also possibly adding extra `if` checks. In my case, I want a `baseurl` locally (`localhost:4000/firstwebsite.com`, `localhost:4000/secondwebsite.com`), but no `baseurl` in production (`firstwebsite.com`, `secondwebsite.com`).

> Are there any other similar conventions that you've seen in Jekyll or another popular plugin that include the `baseurl`?
This uses `absolute_url` instead of `relative_url`, but I think that it still uses `baseurl` under the hood: https://github.com/jekyll/jekyll-feed/blob/f321e1cd8115599340fb3668edab1ae768c25927/lib/jekyll-feed/meta-tag.rb#L19

Reply:
> Aren't you currently having to use `relative_url` or `absolute_url` or at least `prepend: site.baseurl` in your layout to achieve this with paths to links, js, css, images, etc?

No, probably due to
> The `link` and `post_url` tags no longer need `site.baseurl` prepended every time they’re used. Those tags now use our `relative_url` filter to take care of this for you.

from https://jekyllrb.com/news/2019/08/20/jekyll-4-0-0-released/#super-powered-content-transformations-.

But I just realized that my second point in my first reply doesn't make sense, because, similarly to how this plugin can use `relative_url` function regardless of the `baseurl` value, and still work, I can also use the `relative_url` function in my template in both cases as well. My first point, "you have to know to do this" still stands, but as you said, could be fixed by a documentation update.

> Ah, that's a helpful find, but it's also a `tag`, not a `filter`. Jekyll does include the `baseurl` in some `tags`, like `link` and `post_url`, but I'm still not aware of any `filters` that do so.

Ah, I see. I will revert this change and modify the README file tonight.